src/components/button/templates/component/config.js

```diff
@@ -41,25 +53,34 @@ export let buttonConfig = {
         tagline: true,
         tagkey:  'later_tag',
 
-        colors: [ 'creditblue' ],
-
-        logoColors:  {
-            creditblue: 'white'
-        },
+        colors: [ 'creditblue','black'],
 
         allowFundingIcons: false,
 
         defaultColor: 'creditblue'
     },
 
     buynow: {
-        tagline: false,
+        tagline: true,
         tagkey:  'safer_tag',
 
         defaultBranding:     undefined,
         defaultFundingIcons: true,
 
         allowUnbranded: true
+    },
+
+    generic: {
+        tagline: true,
+        tagkey: 'safer_tag',
+        label: '${pp}${paypal}'
+
+    },
+
+    dual: {
```

Comment:
I think we shouldn't have a `dual` option here since it's not a new label. Could we have a `dual_tagline` option for the other labels instead?